skillmap/src/App.tsx

```diff
@@ -66,19 +66,63 @@ interface AppState {
     error?: string;
 }
 
+interface ReadyResources {
```

Comment:
(for later/when making this change on master) could this class be moved out of app.tsx?

Reply:
That might make sense. It really is just an api to the editor that isn't available right away.